Dockerfile

```diff
@@ -4,7 +4,8 @@ WORKDIR /sdk
 COPY ./requirements.txt /sdk
 
 RUN pip install --upgrade pip \
-    && pip install pytest -r requirements.txt
+    && pip install  $(grep -ivE "black" requirements.txt) 
+# black is not available in python2.7 container
```

Comment:
why not create a separate `requirements.txt` for Python 2.7?

Reply:
or, better yet, use conditionals https://stackoverflow.com/a/35614580

Reply:
Changed the approach based on your last suggestion. Thanks!